For musl libc the stackBottom for the main thread is recalculated
just as it is for Android. See: https://bugreports.qt.io/browse/QTBUG-44268

--- qtdeclarative/src/qml/jsruntime/qv4engine.cpp	2016-02-26 10:34:34.000000000 +0100
+++ qtdeclarative/src/qml/jsruntime/qv4engine.cpp	2016-04-04 21:52:43.780823136 +0200
@@ -94,6 +94,11 @@
 #include <valgrind/memcheck.h>
 #endif
 
+#if defined(Q_OS_LINUX) && !defined(__GLIBC__)
+#include <sys/syscall.h>
+#define gettid() syscall(SYS_gettid)
+#endif
+
 QT_BEGIN_NAMESPACE
 
 using namespace QV4;
